concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -71,7 +71,9 @@ EOF
 generate_manifest_file() {
   # This exists because concourse does not support boolean value interpolation by design
   enable_auto_deploy=$([ "${ENABLE_AUTO_DEPLOY:-}" ] && echo "true" || echo "false")
+  [ "${DISABLE_CONTINUOUS_SMOKE_TESTS}" = "true" ] && continuous_smoke_tests_trigger="false" || continuous_smoke_tests_trigger="true"
```

Comment:
RE: accepting any value, I'm not sure what would be the least surprising behaviour. I don't really mind either way.

It could still be written as follows to match the existing style though:
```
continuous_smoke_tests_trigger=$([ "${DISABLE_CONTINUOUS_SMOKE_TESTS}" = "true" ] && echo "false" || echo "true")
```